cirq/ops/common_gates.py

```diff
@@ -524,11 +524,11 @@ def _with_exponent(self,
         return HGate(half_turns=exponent)
 
     def _eigen_components(self):
-        component0 = np.array([[(3 + 2 * np.sqrt(2)), (1 + np.sqrt(2))],
-                               [(1 + np.sqrt(2)), (1)]]) / 2 * (2 + np.sqrt(2))
+        component0 = np.divide(np.array([[(3 + 2 * np.sqrt(2)), (1 + np.sqrt(2))],
```

Comment:
You can just divide with the binary operator.

Reply:
Dividing by binary operator is what I originally wrote, but it doesn't give the correct matrices.

```
component0 = (np.array([[(3 + 2 * np.sqrt(2)),
                                          (1 + np.sqrt(2))],
                                         [(1 + np.sqrt(2)),
                                          (1)]])) / 2 * (2 + np.sqrt(2))

        component1 = (np.array([[(3 - 2 * np.sqrt(2)),
                                          (1 - np.sqrt(2))],
                                         [(1 - np.sqrt(2)),
                                          (1)]])) / 2 * (2 - np.sqrt(2))
```